package.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "scripts": {
         "build": "webpack --env production",
-        "start": "webpack-dev-server --env development",
+        "start": "PUBLIC_CONFIG_PATH=config/config.public.json INFRA_CONFIG_PATH=config/config.infra.json webpack-dev-server --env development",
```

Comment:
why have the env vars been re-added? its in the make file

Reply:
Good catch. It is new branch because the yarn lock went wild. Addressing.